test/functional/services/common/browser.ts

```diff
@@ -163,6 +163,14 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
       return currentWithoutTime;
     }
 
+    /**
+     * Gets the page/document title of the focused window/frame.
+     * https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/chrome_exports_Driver.html#getTitle
+     */
+    public async getTitle() {
+      return await driver.getTitle();
+    }
```

Comment:
This was added courtesy of @dmlemeshko in Slack.

Dzmitry - it looks like we need a required approval from kibana-qa due to touching this file - would super appreciate it if you wouldn't mind doing so! 🙇‍♀️